tests/test_odl.py

```diff
@@ -134,6 +136,44 @@ def test_checkin_success(self):
         eq_(7, self.pool.licenses_available)
         eq_(0, self._db.query(Loan).count())
 
+    def test_checkin_success_with_holds_queue(self):
```

Comment:
I found this test and the next one difficult to get into; some narrative comments would help.